nums/numpy/api/utility.py

```diff
@@ -0,0 +1,181 @@
+# coding=utf-8
+# Copyright (C) 2020 NumS Development Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable = redefined-builtin, too-many-lines, W1401, W0401, W0614
```

Comment:
Nit: is there a reason why some of the disables use the error names and some use the error codes? It might be more readable if you only use erro rnames

Reply:
Done. Did not know that was a feature.